xml/smtp_banners.xml

```diff
@@ -1636,7 +1636,16 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="0" name="service.product" value="Perl"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
-  <fingerprint pattern="^([^ ]+) E?SMTP(?: (?:Service )?Ready\.?)?$" flags="REG_ICASE">
+<fingerprint pattern="Cellopoint E-mail Firewall (v\d+.\d+.\d+) Build (\d+) ready">
```

Comment:
I'd remove the capturing of the `v`, since that really isn't part of the version.